content/about/careers/openff/_index.md

```diff
@@ -5,6 +5,52 @@ weight: 1
+- understands quantum chemistry methods and physical organic chemistry,
+- can help us make decisions about how we generate the training sets for our force fields,
+- will enjoy doing fitting experiments to explore the best approaches to generate superior new force fields
+- is passionate about open source software and open science. 
+
+[To apply, use this form.](https://forms.gle/mXg5bgqsm8KNByZJ9)
+
+The candidate will help lead our efforts around the use of quantum chemical data in generating OpenFF force fields. This includes the generation, curation and management of quantum chemistry datasets. Our QC data is publicly accessible on the MolSSI [QCArchive](https://qcarchive.molssi.org/), which enables computing and storing of quantum chemistry data. It is powered by modular and open source infrastructure, and the candidate would be expected to work closely with scientists working on QCArchive. Depending on the candidate’s interests, the position could also potentially include contributions to QCArchive infrastructure development and maintenance. We also expect the candidate to assist with force field optimization processes and benchmarking studies, perform scientific studies to inform future decisions in force field development, and of course, work together with the rest of our team to iteratively improve our force fields and infrastructure. The candidate will have a reasonable amount of independence in their work, especially in their core area of expertise, although a certain level of coordination is required in this collaborative project.
+
+Many aspects of this position can be tailored around the candidate’s expertise and ambitions and we would encourage everyone excited about applications of quantum chemistry, force field development and open science to apply!
+
+**About us**: We are a distributed team of PhD students, postdocs and software scientists led by [5 academic PIs -- Mike Gilson, David Mobley, Lee-Ping Wang, Michael Shirts, and John Chodera](https://openforcefield.org/about/team/), with more direct leadership from technical lead Jeff Wagner (OMSF) and science lead Lily Wang (OMSF). The effort has been primarily funded by the Open Force Field Consortium and an NIH grant focused on development of accurate biomolecular force fields. The [Open Force Field Consortium](https://openforcefield.org/about/organization/) is an [alliance of pharmaceutical and biotech companies](https://openforcefield.org/about/organization/#open-force-field-consortium) aiming to advance the state of force fields for biomolecular simulation and design through an open collaborative effort. In our organization, the postdocs and PhD students do research and exploratory work using our tools/data, while a team of software and staff scientists (holding PhDs in biomolecular research) work on taking new innovations to production level via high quality professional software and force fields. 
```

Comment:
Lee-Ping Wang is listed here as a site, do we want that?

Reply:
At this point yes, will remove if he doesn't want to be listed (we are discussing).